packages/twenty-server/src/engine/core-modules/open-api/utils/responses.utils.ts

```diff
@@ -298,3 +298,52 @@ export const getJsonResponse = () => {
     },
   };
 };
+
+export const getFindDuplicatesResponse200 = (
+  item: Pick<ObjectMetadataEntity, 'nameSingular'>,
+) => {
+  return {
+    description: 'Successful operation',
+    content: {
+      'application/json': {
+        schema: {
+          type: 'object',
+          properties: {
+            companyDuplicates: {
```

Comment:
missing data key
Think data should be an array here, contrary to the other endpoints